AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -282,212 +288,222 @@ - (void)setNeedsDisplay
       [rasterizedContainerNode setNeedsDisplay];
     });
   } else {
-    // If not rasterized (and therefore we certainly have a view or layer),
-    // Send the message to the view/layer first, as scheduleNodeForDisplay may call -displayIfNeeded.
```

Comment:
Bring this comment block back here, and remove it from the place that it was added to.